### ISSUE TITLE
Filtering in Trips

### ISSUE THREAD

**Issue by janpanchalingam1** [unicef/etools-issues#268]
A few filtering issues observed:
1. After you filter and then refresh your page, the filter fields disappear

2. Filtering by month is not working after we tested in production (April 12, 2017)
Once the filter is applied for months it will show the correct results only once.
If I apply a second filter then the results from the first search will not change.
i.e. I apply September and I will see the results from April. This needs a new card and a new solution.

3. Filtering by results is not working

NOTE: While it has been difficult to reproduce, there are times when filtering in general does not work after the 2nd try (or after refresh)...please ensure proper tests are run on trip, invoice, and action point list filtering

**Comment by janpanchalingam1** [comment]
![image](https://cloud.githubusercontent.com/assets/26144740/24969259/1f7ca394-1f7e-11e7-812c-95d45c0a1027.png)

**Comment by janpanchalingam1** [comment]
![image](https://cloud.githubusercontent.com/assets/26144740/24969284/3ce0a368-1f7e-11e7-8992-0dd614be98c2.png)

**Comment by janpanchalingam1** [comment]
Pass
